po/de.po

```diff
@@ -1388,7 +1388,7 @@ msgstr "RSS abonnieren"
 
 #: externals/header.phtml:4
 msgid "Support the GnuCash project with a donation"
-msgstr "Das GnuCash Projekt mit einer Spende unterstützen."
+msgstr "Das GnuCash Projekt mit einer Spende unterstützen"
```

Comment:
It is a general question: Should tooltips of buttons be full sentences or should we probably apply the ellipses rule?

Reply:
I think the tooltips are mainly to be understood as short hints and do not need punctuation marks. Unless two or more sentences are written.
The main thing is to remain consistent.